src/i_input.c

```diff
@@ -376,69 +376,63 @@ void I_DelayEvent(void)
 // exceed the value of mouse_acceleration_threshold, they are multiplied by
 // mouse_acceleration to increase the speed.
 
+#define MAX_EVENTS 8192
+#define M_MIN (10 << FRACBITS)
+static fixed_t m_accel;
+static fixed_t m_thresh;
 int mouse_acceleration;
 int mouse_acceleration_threshold;
 
-static int AccelerateMouse(int val)
+void I_UpdateMouseAccel(void)
+{
+    m_accel = mouse_acceleration << FRACBITS;
+    m_thresh = mouse_acceleration_threshold << FRACBITS;
```

Comment:
If you want, we can just use `float` in `i_*` files, we religiously stick to fixed math only in render and playsim 😄.

Reply:
Ah, thanks I think float makes sense too. I'll take a closer look later.

Reply:
Fixed by https://github.com/fabiangreffrath/woof/pull/1312/commits/1512264187cb23fcbabf464caf7e91c21a0acc04